guides/v3.7.0/reference/syntax-conversion-guide.md

```diff
@@ -1,24 +1,27 @@
 ## Angle Bracket Syntax
 
-The [Angle Bracket Syntax](https://github.com/emberjs/rfcs/blob/master/text/0311-angle-bracket-invocation.md) is an alternative style of invoking components in templates. The difference between the Angle Bracket Syntax and the Classic Invocation Syntax is purely syntactical and does not affect the semantics of invoking a component.
+The [angle bracket invocation syntax (ABIS)](https://github.com/emberjs/rfcs/blob/master/text/0311-angle-bracket-invocation.md) is an alternative style of invoking components in templates.
```

Comment:
I'm pretty sure it's the convention to capitalise the letters of a phrase you're about to create an abbreviation from 🤔 so that would be: 

```suggestion
The [Angle Bracket Invocation Syntax (ABIS)](https://github.com/emberjs/rfcs/blob/master/text/0311-angle-bracket-invocation.md) is an alternative style of invoking components in templates.
```

Reply:
I am going the opposite way and not use abbreviations at all. Coming back to the material and reading it, I don't think it is improved by the use of abbvs.

Reply:
Yes I agree 👍